apps/zoomcallstatus/zoom_call_status.star

```diff
@@ -0,0 +1,105 @@
+""")
+
+def get_schema():
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Text(
+                id = "email",
+                name = "Zoom User Email",
+                desc = "Your personal zoom email.",
+                icon = "user",
+            ),
+            schema.Text(
+                id = "token",
+                name = "Zoom JWT Token",
+                desc = "Please create a JWT Zoom app and paste your JWT token here.",
+                icon = "key",
+            ),
+        ],
+    )
+
+def show_warning(message):
+    return render.Root(
+        child = center_children(
+            [
+                render.WrappedText(message),
+            ],
+        ),
+    )
+
+def get_user(expected_email, token):
+    res = http.get(PAGED_URL, headers = {"Authorization": "Bearer " + token})
+    if res.status_code != 200:
+        return "Invalid token"
+    foundUser = cache.get("status")
```

Comment:
The cache is shared across all users and installations of the app, so whenever the `"status"` key is present in cache all users will see the same result. I think the best way to deal with that is to include the JWT in the cache key. That way it a) becomes unique for each user and b) is only accessible for users who have access to the JWT.

If the JWT is large, it could be convenient to [hash it](https://github.com/qri-io/starlib/tree/master/hash).